unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -24,6 +24,7 @@ export class CreatorLocks extends React.Component {
   render() {
     const { locks } = this.props
     const { showDashboardForm } = this.state
+    let lockFeed = Object.values(locks).reverse() // We want to display newer locks first
```

Comment:
Is the order right now based on the creation date?

Reply:
Unfortunately we don't seem to have access to creation dates for locks or transactions - but the effect is that they're retrieved in created order. This may not always be the case, so some kind of stored created date would be useful.